### DISCUSSION TITLE
vim-ai plugin not working

### DISCUSSION THREAD

**Discussion by Loungagna** [NvChad/NvChad#1990] [Q&A] [answered]
I have:
```
{
    "madox2/vim-ai",
    lazy = false,
    version = false,
    config = function()
      require("vim-ai").setup()
    end,
  },
```
in plugins/init.lua

Nevertheless, I get a E492: Not an editor command when I  :AI

:Lazy shows vim-ai not loaded (but installed and fetch for updates if asked - no updates needed since it's the latest).

What am I missing?

P.S. :checkhealth shows everything green/OK for NVChad (vim-ai has not check health function)

**Comment by Loungagna** [comment]
Python 3 support is required for the plugin to work.

**Reply by siduck** [reply, marked as answer]
@Loungagna  https://github.com/NvChad/NvChad/issues/1417
